Sources/Penny/+String.swift

```diff
@@ -34,14 +34,21 @@ extension String {
 
 /// `StringProtocol` is basically either `String` or `Substring`.
 extension StringProtocol {
-    /// trims whitespaces and makes the string case, diacritic and punctuation insensitive.
+    /// Trims whitespaces and makes the string case, diacritic and punctuation insensitive.
     func heavyFolded() -> String {
         self.trimmingCharacters(in: .whitespaces)
             .removingOccurrences(of: .punctuationCharacters)
             .lowercased()
             .folding(options: .diacriticInsensitive, locale: nil)
     }
-    
+
+    /// No whitespaces or lines and makes the string case, diacritic and punctuation insensitive.
+    func superHeavyFolded() -> String {
+        self.removingOccurrences(of: .whitespacesAndNewLinesAndPunctuations)
+            .lowercased()
+            .folding(options: .diacriticInsensitive, locale: nil)
```

Comment:
Ew. Don't use `CharacterSet` if you don't have to.
```suggestion
        self.filter { !($0.isWhitespace || $0.isNewline || $0.isPunctuation }
            .lowercased()
            .folding(options: .diacriticInsensitive, locale: nil)
```

Reply:
why though? :eyes: